Frontend/CO.CDP.OrganisationApp/Constants/Country.cs

```diff
@@ -208,47 +170,40 @@ public static class Country
             {"SB", "Solomon Islands"},
             {"SO", "Somalia"},
             {"ZA", "South Africa"},
-            {"GS", "South Georgia and the South Sandwich Islands"},
+            {"KR", "South Korea"},
+            {"SS", "South Sudan"},
             {"ES", "Spain"},
             {"LK", "Sri Lanka"},
             {"SD", "Sudan"},
             {"SR", "Suriname"},
-            {"SJ", "Svalbard and Jan Mayen"},
-            {"SZ", "Swaziland"},
             {"SE", "Sweden"},
             {"CH", "Switzerland"},
-            {"SY", "Syrian Arab Republic"},
-            {"TW", "Taiwan"},
+            {"SY", "Syria"},
             {"TJ", "Tajikistan"},
-            {"TZ", "Tanzania, United Republic of"},
+            {"TZ", "Tanzania"},
             {"TH", "Thailand"},
-            {"TL", "Timor-Leste"},
+            {"BS", "The Bahamas"},
+            {"GM", "The Gambia"},
             {"TG", "Togo"},
-            {"TK", "Tokelau"},
             {"TO", "Tonga"},
             {"TT", "Trinidad and Tobago"},
             {"TN", "Tunisia"},
             {"TR", "Turkey"},
             {"TM", "Turkmenistan"},
-            {"TC", "Turks and Caicos Islands"},
             {"TV", "Tuvalu"},
             {"UG", "Uganda"},
             {"UA", "Ukraine"},
-            {"AE", "United Arab Emirates"},
-            {"US", "United States of America"},
-            {"UM", "United States Minor Outlying Islands"},
+            {"AE", "United Arab Emirates"},            
+            {"US", "United States"},
             {"UY", "Uruguay"},
             {"UZ", "Uzbekistan"},
             {"VU", "Vanuatu"},
+            {"VA", "Vatican City"},
             {"VE", "Venezuela"},
             {"VN", "Vietnam"},
-            {"VG", "Virgin Islands, British"},
-            {"VI", "Virgin Islands, U.S."},
-            {"WF", "Wallis and Futuna"},
-            {"EH", "Western Sahara"},
             {"YE", "Yemen"},
             {"ZM", "Zambia"},
-            {"ZN", "Zimbabwe"}
+            {"ZW", "Zimbabwe"}
```

Comment:
Does changing this country list affect any other functionality not related to this PR? If yes, is it intended?

Reply:
I have updated from the list provided by Sam, and looks like earlier there was mistake which is corrected now.